test/common/util.ts

```diff
@@ -51,6 +52,9 @@ export function $x(
  */
 export var selectors = {
     nav: "#stage > #nav",
+    modal: "#stage",
     tab: "//hx-tablist/hx-tab",
     visreg: "*[data-visreg]",
+    openModal: "//div/div/main/section/div/hx-disclosure",
+    closeModal: "//div/div/main/section/div/hx-modal/div/button",
```

Comment:
Please don't right click elements in the dev console and choose "copy XPath selector", it uses *really* absolute dom node locations. Besides being unreadable, are also more brittle!

I think you should use these xpath locators as a starting point, and try to shorten them to the smallest query possible to return the correct element.

Reply:
Reduced the xpath value.